app/addons/databases/actions.js

```diff
@@ -0,0 +1,105 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+define([
+  'app',
+  'api',
+  'addons/databases/stores',
+  'addons/databases/actiontypes',
+  'addons/databases/resources'
+],
+function (app, FauxtonAPI, Stores, ActionTypes, Resources) {
+  return {
+    init: function (collection, backboneCollection) {
+      FauxtonAPI.dispatch({
+        type: ActionTypes.DATABASES_INIT,
+        options: {
+          collection: collection,
+          backboneCollection: backboneCollection
+        }
+      });
+    },
+
+    setPage: function (_page) {
+      FauxtonAPI.dispatch({
+        type: ActionTypes.DATABASES_SETPAGE,
+        options: {
+          page: _page
+        }
+      });
+    },
+
+    setStartLoading: function () {
+      FauxtonAPI.dispatch({
+        type: ActionTypes.DATABASES_STARTLOADING,
+        options: {}
+      });
+    },
+
+    setLoadComplete: function () {
+      FauxtonAPI.dispatch({
+        type: ActionTypes.DATABASES_LOADCOMPLETE,
+        options: {}
```

Comment:
It depends on whether we just always want to have options - if we do it case-by-case we can just kick them out.